ccm_web/server/localTypes/express-session/index.d.ts

```diff
@@ -0,0 +1,12 @@
+import 'express-session'
```

Comment:
Does this file need to be added to the repo if `"@types/express-session": "^1.17.3"` is specified in `package.json`?  I thought that adding that module dependency would give all the type info necessary.

Reply:
This is an instance of [declaration merging](https://www.typescriptlang.org/docs/handbook/declaration-merging.html). Basically, you can store arbitrary data in a session, but in TypeScript, you need to define the shape of that data. This file is adding those types.

See also https://github.com/expressjs/session/issues/799